### ISSUE TITLE
question about training datasets

### ISSUE THREAD

**Issue by MASTERCHIEF343** [sunghoonim/DPSNet#22]

**Comment by MASTERCHIEF343** [comment]
Hi, I'm working on my network.And i use rgbd and mvs datasets for training.But I find that mvs isn't work very well, the test accuracy is bad.But the training loss is very well and accuracy for rgbd test datasets is samilliary with you.Have you ever met that situation?
